### DISCUSSION TITLE
extend Flows Trigger: "Event Hook" of type "Filter(Blocking)" to cancel Events

### DISCUSSION THREAD

**Discussion by funkhaus-phil** [directus/directus#14316] [Feature Requests]
While fiddling around with the new flows i was missing an option to cancel the chosen event when type Filter(Blocking) is picked on an event hook.
Of course we are able to write a custom hook when this behaviour is intended, but since the Flow hints in that direction, it would be great to quickly realise some logic with said flows and interrupt for example item creation under specific circumstances.

I also figured that is already possible by creating an error while triggering the flow, but the following points speak against doing so:
1. creating an actual error is not that easy, since flows are evaluated pretty safe
2. it's unsure if the error is persistent when updating directus
3. no proper error message on interruption of the event

I'd like to take the chance and thank you for all the great work on this Project <3

**Comment by nickrum** [comment]
This will be possible once #15283 has been released.
